src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+    list-style: none;
+  }
+
+  &__item:last-of-type {
+    margin: 0;
+  }
+
+  &__link {
+    display: inline-block;
+    color: $darkblue;
+    font-weight: 500;
+    text-decoration: none;
+    line-height: 60px;
+    text-transform: uppercase;
+    transition: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated